stdlib/2and3/zipfile.pyi

```diff
@@ -141,6 +141,25 @@ class ZipInfo:
         def is_dir(self) -> bool: ...
     def FileHeader(self, zip64: Optional[bool] = ...) -> bytes: ...
 
+if sys.version_info >= (3, 8):
+    class Path:
+        name: str
```

Comment:
In the code (as opposed to the docs), this is a property, maybe it's better to annotate it as such so the type-checker doesn't allow to set it?